monai/handlers/stats_handler.py

```diff
@@ -133,16 +133,16 @@ def iteration_completed(self, engine: Engine) -> None:
     def exception_raised(self, engine: Engine, e: Exception) -> None:
         """
         Handler for train or validation/evaluation exception raised Event.
-        Print the exception information and traceback.
+        Print the exception information and traceback. This callback may be skipped because the logic
+        with ignite can only trigger the first attached handler for `EXCEPTION_RAISED` event.
 
         Args:
             engine: Ignite Engine, it can be a trainer, validator or evaluator.
             e: the exception caught in Ignite during engine.run().
 
         """
         self.logger.exception(f"Exception: {e}")
-        # import traceback
-        # traceback.print_exc()
+        raise e
```

Comment:
Codecov is complaining about line 145 not being covered in the tests, maybe we should add an exception raise test somewhere?

Reply:
added new tests in the latest commit.
Thanks.